gerritlab/global_vars.py

```diff
@@ -122,7 +122,7 @@ def _parse_remote_url(url: str):
     if m:
         url = "https://" + m[1]
         path = m[2]
-    elif re.match(r"https?://.*", url):
+    elif re.match(r"^(https|ssh)?://.*", url):
```

Comment:
```suggestion
    elif re.match(r"^(?:https?|ssh)://.*", url):
```

Reply:
Use a non-capturing group.. and make sure the `?` applies to the `s` in `https` so that both http and https are matched.

Reply:
Thanks for pointing that out!